ui-tests/cypress/support/controllers/cmsActions/ContentTypesAdminConsoleController.js

```diff
@@ -0,0 +1,19 @@
+import AbstractController from '../abstractController';
+
+import {contentTypesURL, contentTypesJacmsURL} from '../controllersEndPoints';
```

Comment:
please, split the two controllers in two different files

Reply:
Done